content/docs/sidebar.yaml

```diff
@@ -228,6 +228,8 @@
       slug: extensions/pg_embedding
```

Comment:
Should this slug still reference pg_embedding?

Reply:
We still have the pg_embedding docs there with a warning. We can remove it sometime in the near future.